scripts/zones/The_Boyahda_Tree/npcs/Mandragora_Warden.lua

```diff
@@ -21,8 +21,10 @@ end
 function onTrigger(player,npc)
     if player:getVar("MissionStatus") == 4 then
         local dialog = player:getVar("mandialog")
-
-        if dialog == 0 then
+        if player:hasItem(1181) then
+            player:startEvent(10)
+            player:PrintToPlayer("The warden appears to want something from you...")
```

Comment:
Print to player? If its supose to day something it will be an event or a textid

Reply:
Hi @KnowOne134 😸

I know, I just couldn't find the textid for any of the PrintToPlayer that already existed for this in:
 
https://github.com/KnowOne134/FFXI_Events/blob/master/the_boyahda_tree.json
😿 not even for the one that I added. (Can be seen in https://www.youtube.com/watch?v=L1TGPHzvwzg)

Reply:
Could you use a timestamped link please? it is a very long video to be scrolling around in.

It is likely a cs event and possibly requires specific parameters passed to said cs event. in any case, printToPlayer to fake it is not something we do in the project. That function is mainly for debugging and for people to make their custom things out of.

Reply:
```
  <thing type="DialogTableEntry">
    <field name="index">7333</field>
    <field name="text">≺Possible Special Code: 1F≻≺Possible Special Code: 0F≻The warden appears to want something from you...≺Prompt≻</field>
  </thing>
```

Reply:
Hi @KnowOne134 :

Changed it to use messageText and messageSpecial, and added the text to IDs.lua

@TeoTwawki :

This is a video capture from @Wiggo32 😸  https://youtu.be/7GekeM7HOUU?t=2513

@ibm2431  thank you for the info 😺

I am always thankful for your help. Can't wait for the mandragora's to be lively in oldschool server